rabbitmq-ballerina/client.bal

```diff
@@ -149,57 +153,57 @@ public client class Client {
     # Deletes the queue with the given name although it is in use or has messages in it.
     # If the `ifUnused` or `ifEmpty` parameters are given, the queue is checked before deleting.
     # ```ballerina
-    # rabbitmq:Error? deleteResult = newClient->queueDelete("MyQueue");
+    # check rabbitmqClient->queueDelete("MyQueue");
     # ```
     #
-    # + queueName - Name of the queue to be deleted
+    # + queueName - The name of the queue to be deleted
     # + ifUnused - True if the queue should be deleted only if it's not in use
     # + ifEmpty - True if the queue should be deleted only if it's empty
-    # + return - A `rabbitmq:Error` if an I/O error is encountered or else `()`
+    # + return - A `rabbitmq:Error` if an I/O error occurred or else `()`
     isolated remote function queueDelete(string queueName, boolean ifUnused = false, boolean ifEmpty = false)
                         returns Error? {
         return nativeQueueDelete(queueName, ifUnused, ifEmpty, self.amqpChannel);
     }
 
     # Deletes the exchange with the given name.
     # ```ballerina
-    # rabbitmq:Error? deleteResult = newClient->exchangeDelete("MyExchange");
+    # check rabbitmqClient->exchangeDelete("MyExchange");
     # ```
     #
     # + exchangeName - The name of the exchange
-    # + return - A `rabbitmq:Error` if an I/O error is encountered or else `()`
+    # + return - A `rabbitmq:Error` if an I/O error occurred or else `()`
     isolated remote function exchangeDelete(string exchangeName) returns Error? {
         return nativeExchangeDelete(exchangeName, self.amqpChannel);
     }
 
     # Purges the content of the given queue.
     # ```ballerina
-    # rabbitmq:Error? purgeResult = newClient->queuePurge("MyQueue");
+    # check rabbitmqClient->queuePurge("MyQueue");
     # ```
     #
     # + queueName - The name of the queue
-    # + return - A `rabbitmq:Error` if an I/O error is encountered or else `()`
+    # + return - A `rabbitmq:Error` if an I/O error occurred or else `()`
     isolated remote function queuePurge(string queueName) returns Error? {
         return nativeQueuePurge(queueName, self.amqpChannel);
     }
 
     # Closes the `rabbitmq:Client`.
     # ```ballerina
-    # rabbitmq:Error? closeResult = newClient.close();
+    # check rabbitmqClient.close();
     # ```
     #
     # + closeCode - The close code (for information, go to the "Reply Codes" section in the
     #               [AMQP 0-9-1 specification] (#https://www.rabbitmq.com/resources/specs/amqp0-9-1.pdf))
```

Comment:
```suggestion
    #               (#https://www.rabbitmq.com/resources/specs/amqp0-9-1.pdf))
```
Please check if the link is added correctly.

Reply:
Link verified